### DISCUSSION TITLE
Request: Add GNOME DE Wallpapers

### DISCUSSION THREAD

**Discussion by sapphired18** [t1m0thyj/WinDynamicDesktop#615] [Ideas]
Adding this might be cool, GNOME has many dual mode wallpapers
https://zebreus.github.io/all-gnome-backgrounds/

**Comment by t1m0thyj** [comment]
I agree this would be cool but don't have time to build all the DDW files myself. If someone does (or better yet if someone creates a script to build them), I'd be happy to add them to the WinDD themes website 🙂

**Comment by tjsuominen** [comment]
Any news, they would look great!
